### DISCUSSION TITLE
Does Magick.Net library supports SWF files ?

### DISCUSSION THREAD

**Discussion by NullException1** [dlemstra/Magick.NET#606] [General]
Hi,
I want to know whether Magick.Net supports SWF files. Whenever I am trying to read an SWF file, Magick.Net throws exception saying "No decode delegate for this format". Also, I browsed various articles and found **imagemagick** itself does not support SWF files. If that is the case, can anybody has any idea about how we can read/write/convert/resize SWF files in Magick.Net ?

Thank you !!

cc : @dlemstra

**Comment by dlemstra** [comment]
There is no support for reading SWF files at the moment. We might be able to add support for this with the help of the format specifications. But even if we find them it will probably not be added anytime soon.
